docs/user/exercises/quiz.rst

```diff
@@ -293,20 +371,27 @@ Resolve a quiz exercise
     - The quiz page is compose by:
 
         - Number and title of the question
+
         - Points for solving that question
+
         - The quiz statement
 
         - Options:
 
             - Options with circles means one choice could be correct.
+
             - Options with squares means multiple options could be correct.
 
         - In the footer:
 
-            - Number of questions and overall points
-            - Time left to complete the quiz
+            - Number of questions and overall points.
+
+            - Time left to complete the quiz.
+
             - Last time saved: The quiz will save all changes after they occur.
-            - Connection status
+
+            - Connection status.
+
             - Submit button: To allow student submit the quiz before the time ends.
 
         - In case of Drag and Drop questions, the items to be drag and drop in the option spots will be available in the right side.
```

Comment:
In "the" case of Drag and Drop questions, the items to be "dragged" and "dropped" in the option spots will be available "on" the right side.